src/html/transfers.html

```diff
@@ -7,9 +7,38 @@ <h1>Your Transfers</h1>
     </button>
     <!-- TODO: <nav> -->
   </header>
+  <div class="recovery-container">
+    <button class="link" data-behavior="toggleRecovery">Recover with deposit tx hash</button>
```

Comment:
A design question: "deposit tx hash" does not indicate "Ethereum lock transaction hash OR NEAR withdraw transaction hash" to me. Not sure what it should say instead. Can wait to fix this later though.

Reply:
This will be improved with the new design. But the tx hash input field can accept both an Ethereum or Near tx hash.